### PR TITLE
can u check this!

### PR DESCRIPTION
**Pull request by awualex**
,,,,

### CODE REVIEW

File: Class-Code/Elementary-CA/awu/sketch.js [hunk 1]
```diff
@@ -1,14 +1,103 @@
-//Your Name(s)
+//Your Name(s) Alex Wu
 //Project Name
```
Comment (Mr-Darfler) [inline Class-Code/Elementary-CA/awu/sketch.js]: Make Sure you Add in the project name and Date

File: Class-Code/Elementary-CA/awu/sketch.js [hunk 2]
```diff
@@ -1,14 +1,103 @@
...
+                noStroke();
+                fill(0);
+                rect(i * cell_size, k * cell_size, cell_size, cell_size)
+            }
+        }
+    }
+
+}
+
+function ruleset(uno, dos, tres) {
```
Comment (Mr-Darfler) [inline Class-Code/Elementary-CA/awu/sketch.js]: Naming convention is a bit odd. consider more descriptive  names

File: Class-Code/Elementary-CA/awu/sketch.js [hunk 3]
```diff
@@ -1,14 +1,103 @@
...
+
+/*
+let gen_new
+for(let k = 1; k < 400; k++) {
+    gen_new = Array(list_length).fill(0);
+    gen_new[0]=gen_0[0];
+    gen_new[list_length-1]=gen_0[list_length-1];
+    for(let j =1; j < list_length -1; j++){gen_new[j]=ruleset(gen_0[j-1],gen_0[j],gen_0[j+1])}
+    ca.push(gen_new);
+    gen_0 = gen_new ;}
```
Comment (Mr-Darfler) [inline Class-Code/Elementary-CA/awu/sketch.js]: I don't think you need to re assign gen_0

**Review by Mr-Darfler — COMMENTED** [review]
@awualex I Made some comments. take a look

**Review by Mr-Darfler — CHANGES_REQUESTED** [review]
please change code

**Comment by awualex** [comment]
Now it works ;0

**Review by Mr-Darfler — APPROVED** [review]

**Comment by Mr-Darfler** [comment]
Alex, please 

- remove you commented draw() function
- update your existing draw() function to look more like
```
function draw(){
    display();
    generate();
    scrol();
```
- take the code from your existing draw() function and turn them into the functions above. This will help clean up your code and make it easier to read.

**Comment by Mr-Darfler** [comment]
Closing pull request until final commit

**Pull request closed without merging**